File: android-jsBridgeCore/src/main/java/me/yamlee/jsbridge/ui/DelegateListener.java
package me.yamlee.jsbridge.ui;


/**
 * 代理部分操作监听类
 *
 * @author yamlee
 */
public interface DelegateListener {
    void onClickErrorView();

    void onClickHeaderRight(String clickUri);

    void onClickMoreMenuItem(WebHeader.ListIconTextModel menuItem);
}

File: android-jsBridgeCore/src/main/java/me/yamlee/jsbridge/ui/DelegateListener.kt
package me.yamlee.jsbridge.ui


/**
 * 代理部分操作监听类
 *
 * @author yamlee
 */
interface DelegateListener {
    fun onClickErrorView()

    fun onClickHeaderRight(clickUri: String)

    fun onClickMoreMenuItem(menuItem: WebHeader.ListIconTextModel)
}
